acceptance-tests/src/test/resources/uk/gov/di/test/features/006_registration_journey.feature

```diff
@@ -80,4 +80,17 @@ Feature: Registration Journey
     Then the user is taken to the "You’ve created your GOV.UK One Login" page
     When the user clicks the continue button
     Then the user is returned to the service
-    And the user logs out
\ No newline at end of file
+    And the user logs out
+
+
+  @partialRegUser
+  Scenario: Partial registered user is able to complete registration when they restart journey and select forgotten password for sms user
+    Given the user "TEST_USER_EMAIL_4" is partial registered up to choose how to get security codes page
+    Then the user is taken to the "Choose how to get security codes" page
```

Comment:
Can this line be incorporated into the Given? At the moment it reads Given, Then, When